test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -387,8 +385,10 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 			title
 		);
 
+		// @todo Remove the deprecated query once GB v10.7+ is up on prod. This is needed for now to make sure all tests (more specifically upgrade tests that use non-edge sites) pass.
+		const insertedBlockItemDeprecatedQuery = `.edit-post-layout__inserter-panel .block-editor-block-types-list button.editor-block-list-item-${ prefix }${ blockClass }`;
```

Comment:
Ops, this should have been named "inserter" not "inserted" 🤦🏻 . Good thing is that we'll remove it now that v10.7.0 is on WPCOM.

Reply:
Removed in https://github.com/Automattic/wp-calypso/pull/53395.